src/libraries/System.Private.CoreLib/src/System/Environment.Browser.cs

```diff
@@ -34,5 +34,35 @@ private static OperatingSystem GetOSVersion()
         /// </summary>
         /// <returns>Path of the executable that started the currently executing process</returns>
         private static string? GetProcessPath() => null;
+
+        /// <summary>
+        /// Get the CPU usage, including the process time spent running the application code, the process time spent running the operating system code,
+        /// and the total time spent running both the application and operating system code.
+        /// </summary>
+        [SupportedOSPlatform("maccatalyst")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        public static ProcessCpuUsage CpuUsage
+        {
+            get
+            {
+                Interop.Sys.ProcessCpuInformation cpuInfo = default;
+                Interop.Sys.GetCpuUtilization(ref cpuInfo);
```

Comment:
Is GetCpuUtilization going to work in browser?
https://github.com/dotnet/runtime/blob/67bb10fc3c6f10381a63f4eb87b2753d91740fb0/src/native/libs/System.Native/pal_time.c#L124
(I'm not sure what HOST_BROWSER means here, so maybe it's not related?)

Reply:
I did that because I am seeing 

https://github.com/dotnet/runtime/blob/67bb10fc3c6f10381a63f4eb87b2753d91740fb0/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L2331

which is defined under the `'$(TargetsBrowser)' == 'true' `

Reply:
The API is going to return all zeros for Browser given how `SystemNative_GetCpuUtilization` is implemented, so it is about as unsupported for browser as it is for iOS.

Reply:
In other places, we just depend on `SystemNative_GetCpuUtilization` to not return anything when the capability is unsupported. I think it would be best to do the same here as well - have one implementation that just calls `SystemNative_GetCpuUtilization` in `Environment.UnixOrBrowser.cs` without any ifdefs.

Reply:
Does `Environment.UnixOrBrowser.cs` cover iOS too? or do I need to keep the iOS implementation?

Reply:
Looks not as I am seeing `Environment.UnixOrBrowser.cs` defined under `<ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true' or '$(TargetsWasi)' == 'true'">`

Reply:
Now I see `Environment.iOS.cs` is defined under `<ItemGroup Condition="'$(TargetsUnix)' == 'true'">`. Ok, I think one common implementation in `Environment.UnixOrBrowser.cs` shoudl cover them all, I guess. CI will help with that too.

Reply:
Right, iOS is Unix.